Resources/public/js/views/services/plugins/ez-userdraftsplugin.js

```diff
@@ -0,0 +1,245 @@
+             */
+            this._targetAttributeName = null;
+            /**
+             * Holds a temporary reference to limit of items to be returned in an event target
+             *
+             * @property _targetLimit
+             * @type {Number}
+             * @protected
+             */
+            this._targetLimit = null;
+        },
+
+        /**
+         * Sorts versions by modification date
+         *
+         * @method _sortByModificationDate
+         * @protected
+         * @param a {Object} version hash
+         * @param b {Object} version hash
+         * @return Number
+         */
+        _sortByModificationDate: function (a, b) {
+            return (new Date(b.get('modificationDate'))).getTime() - (new Date(a.get('modificationDate'))).getTime();
+        },
+
+        /**
+         * Collects drafts data
+         *
+         * @method _collectDraftsData
+         * @protected
+         * @param error {Boolean} is error?
+         * @param versions {Array} list of versions
+         * @return {Y.Promise}
+         */
+        _collectDraftsData: function (error, versions) {
+            var host = this.get('host');
+
+            if (error) {
+                host._error(error.message);
+                this._target.set('loadingError', true);
+                this._resetTempVariables();
+
+                return;
+            }
+
+            versions.sort(this._sortByModificationDate);
+            versions.length = this._targetLimit;
+
+            Y.Promise
+                .resolve(versions)
+                .then(Y.bind(this._loadDraftContentInfo, this))
+                .then(Y.bind(this._loadDraftContentType, this))
+                .then(Y.bind(this._fillDraftContentTypeModel, this))
+                .catch(Y.bind(host._error, host));
+        },
+
+        /**
+         * Gets content info for each version
+         *
+         * @method _collectDraftsData
+         * @protected
+         * @param versions {Array} list of versions
+         * @return {Y.Promise}
+         */
+        _loadDraftContentInfo: function (versions) {
+            var drafts = this.get('drafts'),
+                contentService = this.get('host').get('capi').getContentService(),
+                promises = [];
+
+            versions.forEach(function (version) {
+                var contentId = version.get('resources.Content');
+
+                drafts.add(version);
+
+                promises.push(new Y.Promise(Y.bind(function (resolve, reject) {
+                    contentService.loadContentInfo(contentId, Y.bind(function (error, response) {
+                        if (error) {
+                            reject('Cannot load content info: ' + contentId);
+                            this._target.set('loadingError', true);
+                            this._resetTempVariables();
+
+                            return;
+                        }
+
+                        resolve(response.document.Content);
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/591/files#r66587338, don't parse the response here and resolve the Pomise with the ContentInfo model